code/modules/hydroponics/plant_genes.dm

```diff
@@ -187,7 +187,7 @@
 
 /datum/plant_gene/reagent/liquidelectricity
 	name = "Liquid Electricity"
-	reagent_id = /datum/reagent/consumable/liquidelectricity
+	reagent_id = /datum/reagent/consumable/liquidelectricity/enriched
```

Comment:
does enriched liquid electricity count as normal liquid electricity for reactions and such? if not, you might be removing some possible botany recipes here (including, say, electrolysis)

Reply:
not removed, just needs different reagents.